src/db.js

```diff
@@ -1,70 +1,169 @@
-import {nanoid} from 'nanoid';
+// const transactionId = transactions.map( (transaction) =>
```

Comment:
Please delete comments with unused code. In general there should be no or just useful comments in your final PR.

Reply:
Thanks for advice. That comments sneaked its way into the db. It´s removed